rotkehlchen/globaldb/handler.py

```diff
@@ -984,6 +986,87 @@ def add_single_historical_price(entry: HistoricalPrice) -> bool:
 
         return True
 
+    @staticmethod
+    def add_manual_current_price(
+            from_asset: Asset,
+            to_asset: Asset,
+            price: Price,
+    ) -> None:
+        """Adds manual current price and turns previously known manual current price into a
```

Comment:
```suggestion
        """
        Adds manual current price and turns previously known manual current price into a
```

Here and below kind of a nitpick for docstrings